rotkehlchen/api/rest.py

```diff
@@ -4038,11 +4040,11 @@ def get_custom_asset_types(self) -> Response:
 
     def get_event_details(self, identifier: int) -> Response:
         dbevents = DBHistoryEvents(self.rotkehlchen.data.db)
-        event = dbevents.get_history_event_by_identifier(identifier=identifier)
+        event = dbevents.get_evm_event_by_identifier(identifier=identifier)
         if event is None:
             return api_response(wrap_in_fail_result('No event found'), status_code=HTTPStatus.NOT_FOUND)  # noqa: E501
 
-        details = event.get_details()
+        details = event.get_details()  # pylint: disable=no-member  # not sure why pylint doesn't see `get_details` here. The type is `EvmEvent`.  # noqa: E501
```

Comment:
I've run the linter and seems that this is not needed. I installed the deps and removed this comment and linter passed